### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by TFiFiE** [gcp/leela-zero-server#169]
Are the learning rates used to train the networks kept track of? Shouldn't these be part of their "signature" like the number of training steps or at least visible somewhere?

**Comment by roy7** [comment]
No record of them has been kept to my knowledge. :(

**Comment by gcp** [comment]
There isn't necessarily a single learning rate per network.
